src/views/components/search.html

```diff
@@ -188,9 +191,9 @@
 			}
 		},
 		scrollToSearch () {
-			window.scrollTo({
-				top: Math.floor(document.querySelector('.search-input').getBoundingClientRect().top / 100) * 100,
-				behavior: 'smooth',
+			document.querySelector('.search-input').scrollIntoView({
```

Comment:
This now scrolls to input in the middle of the page, instead of at the top.

Reply:
I fixed scroll issue.
And accidently used `push -f` git command and restored it again.